app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -182,6 +182,9 @@ private void onFragmentLoaded() {
         }
         searchAdapter.notifyDataSetChanged();
         setListShown(true);
+
+        String query = getArguments().getString(ARG_QUERY);
+        setEmptyText(getString(R.string.no_results_for_query) + " \"" + query + "\"");
```

Comment:
You need to keep in mind that the sentence construction in other languages could be different, so the query string doesn't always show at the end. You should use %1$s in your string on values/strings.xml and pass the query term as an extra argument to getString()